datasets/menyo_20k_mt/menyo_20k_mt.py

```diff
@@ -0,0 +1,111 @@
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""MENYO-20k: A Multi-domain English - Yorùbá Corpus for Machine Translations"""
+
+from __future__ import absolute_import, division, print_function
+
+import csv
+
+import datasets
+
+
+# Find for instance the citation on arxiv or on the dataset repo/website
+_CITATION = """\
+@dataset{david_ifeoluwa_adelani_2020_4297448,
+  author       = {David Ifeoluwa Adelani and
+                  Jesujoba O. Alabi and
+                  Damilola Adebonojo and
+                  Adesina Ayeni and
+                  Mofe Adeyemi and
+                  Ayodele Awokoya},
+  title        = {MENYO-20k: A Multi-domain English - Yorùbá Corpus
+                  for Machine Translation},
+  month        = nov,
+  year         = 2020,
+  publisher    = {Zenodo},
+  version      = {1.0},
+  doi          = {10.5281/zenodo.4297448},
+  url          = {https://doi.org/10.5281/zenodo.4297448}
+}
+"""
+
+
+# You can copy an official description
+_DESCRIPTION = """\
+MENYO-20k is a multi-domain parallel dataset with texts obtained from news articles, ted talks, movie transcripts, radio transcripts, science and technology texts, and other short articles curated from the web and professional translators. The dataset has 20,100 parallel sentences split into 10,070 training sentences, 3,397 development sentences, and 6,633 test sentences (3,419 multi-domain, 1,714 news domain, and 1,500 ted talks speech transcript domain). The development and test sets are available upon request.
+"""
+
+
+_HOMEPAGE = "https://zenodo.org/record/4297448#.X81G7s0zZPY"
+
+
+_LICENSE = "For non-commercial use because some of the data sources like Ted talks and JW news requires permission for commercial use."
+
+
+# The HuggingFace dataset library don't host the datasets but only point to the original files
+# This can be an arbitrary nested dict/list of URLs (see below in `_split_generators` method)
+_URL = "https://github.com/dadelani/menyo-20k_MT/raw/main/data/train.tsv"
+
+
+class Menyo20kMt(datasets.GeneratorBasedBuilder):
+    """MENYO-20k: A Multi-domain English - Yorùbá Corpus for Machine Translations"""
+
+    VERSION = datasets.Version("1.0.0")
+
+    BUILDER_CONFIGS = [
+        datasets.BuilderConfig(
+            name="menyo_20k_mt",
+            version=VERSION,
+            description="MENYO-20k: A Multi-domain English - Yorùbá Corpus for Machine Translations",
+        )
+    ]
+
+    def _info(self):
+
+        return datasets.DatasetInfo(
+            # This is the description that will appear on the datasets page.
+            description=_DESCRIPTION,
+            # This defines the different columns of the dataset and their types
+            features=datasets.Features(
+                {
+                    "en": datasets.Value("string"),
+                    "yor": datasets.Value("string"),
```

Comment:
can you use the `Translation` feature type here ?
you can find an example of usage in [flores](https://github.com/huggingface/datasets/blob/master/datasets/flores/flores.py) for example